source/fundamentals/geo.txt

```diff
@@ -0,0 +1,332 @@
+- Elevation in the third position (optional)
+
+The following is the **position** of the MongoDB Headquarters in
+New York City, NY.
+
+.. code-block:: go
+   :copyable: False
+
+   []float64{-73.986805, 40.7620853}
+
+.. important:: Longitude then Latitude
+
+  GeoJSON orders coordinates as **longitude** first and **latitude** second.
+  This may be surprising as geographic coordinate system conventions generally list
+  latitude first and longitude second. Make sure to check what format any other
+  tools you are working with use. Popular tools such as OpenStreetMap and Google
+  Maps list coordinates as latitude first and longitude second.
+
+
+Types
+`````
+
+Your GeoJSON object's type determines its geometric shape. Geometric shapes are
+made up of positions.
+
+Here are some common GeoJSON types and how you can specify them with positions:
+
+- ``Point``: a single position. This could represent the location of the
+  MongoDB Headquarters.
+
+  .. code-block:: go
+     :copyable: False
+
+     bson.D{
+         {"name", "HQ"},
+         {"location", bson.D{
+             {"type", "Point"},
+             {"coordinates", []float64{-73.986805, 40.7620853}},
+         }},
+     }
+
+- ``LineString``: an array of two or more positions that forms a series of line
+  segments. This could represent
+  `the route of the Great Wall of China <https://commons.wikimedia.org/wiki/File:GreatWallChina4.png>`__.
+
+  .. code-block:: go
+     :copyable: False
+
+     bson.D{
+         {"name", "Great Wall of China"},
+         {"location", bson.D{
+             {"type", "LineString"},
+             {"coordinates", [][]float64{
+                 {116.572, 40.430},
+                 {116.570, 40.434},
+                 {116.567, 40.436},
+                 {116.566, 40.441},
+             }}},
+         },
+     }
+
+- ``Polygon``: an array of positions in which the first and last
+  position are the same and enclose some space. This could represent
+  `the land within Vatican City <https://commons.wikimedia.org/wiki/File:Vatican_City_map_EN.png>`__.
+
+  .. code-block:: go
+     :copyable: False
+
+     bson.D{
+         {"name", "Vatican City"},
+         {"location", bson.D{
+             {"type", "Polygon"},
+             {"coordinates", [][][]float64{{
+                 {116.572, 40.430},
+                 {116.570, 40.434},
+                 {116.567, 40.436},
+                 {116.572, 40.430},
+             }}},
+         }},
+     }
+
+
+To learn more about the shapes you can use in MongoDB, see the
+:manual:`GeoJSON manual entry </reference/geojson/>`.
+
+For definitive information on GeoJSON, see the
+`official IETF specification <https://datatracker.ietf.org/doc/html/rfc7946>`__.
+
+Legacy Coordinate Pairs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can store geospatial data using ``x`` and ``y`` coordinates on
+a two-dimensional Euclidean plane. We refer to coordinates on a two-dimensional
+plane as "legacy coordinate pairs."
```

Comment:
Suggestion: Use standard formatting for introducing a definition

Suggestion (Mike): Consider adding language to distinguish legacy coordinate pair from GeoJSON. 

```suggestion
In addition to using GeoJSON, you can store location data as a **legacy coordinate pair**.
A legacy coordinate pair is an ``x`` and ``y`` value representing a point on
a two-dimensional Euclidean plane.
```